modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/UserCardTag.java

```diff
@@ -0,0 +1,512 @@
+		}
+
+		jspWriter.write("</span>");
+		jspWriter.write("</span>");
+		jspWriter.write("</div>");
+
+		if (_selectable) {
+			jspWriter.write("</label>");
+			jspWriter.write("</div>");
+		}
+
+		jspWriter.write("</div>");
+
+		jspWriter.write("<div class=\"card-body\">");
+		jspWriter.write("<div class=\"card-row\">");
+		jspWriter.write("<div class=\"autofit-col autofit-col-expand\">");
+		jspWriter.write("<p class=\"card-title\">");
+		jspWriter.write("<span class=\"text-truncate-inline\">");
+
+		if (!_disabled && Validator.isNotNull(_href)) {
+			jspWriter.write("<a class=\"text-truncate\" href=\"");
+			jspWriter.write(_href);
+			jspWriter.write("\">");
+			jspWriter.write(_name);
+			jspWriter.write("</a>");
+		}
+		else {
+			jspWriter.write("<span class=\"text-truncate\">");
+			jspWriter.write(_name);
+			jspWriter.write("</span>");
+		}
+
+		jspWriter.write("</span>");
+		jspWriter.write("</p>");
+		jspWriter.write("<p class=\"card-subtitle\">");
+		jspWriter.write("<span class=\"text-truncate-inline\">");
+		jspWriter.write("<span class=\"text-truncate\">");
+		jspWriter.write(_subtitle);
+		jspWriter.write("</span>");
+		jspWriter.write("</span>");
+		jspWriter.write("</p>");
+
+		if (!ListUtil.isEmpty(_labels)) {
+			jspWriter.write("<div class=\"card-detail\">");
+
+			for (LabelItem labelItem : _labels) {
+				jspWriter.write("<span class=\"label ");
+
+				if ((boolean)labelItem.get("dismissible")) {
+					jspWriter.write("label-dismissible ");
+				}
+
+				if ((boolean)labelItem.get("large")) {
+					jspWriter.write("label-lg ");
+				}
+
+				if (Validator.isNotNull(labelItem.get("displayType"))) {
+					jspWriter.write(
+						"label-" + labelItem.get("displayType") + " ");
+				}
+
+				jspWriter.write("\">");
+				jspWriter.write(String.valueOf(labelItem.get("label")));
+				jspWriter.write("</span>");
+			}
+
+			jspWriter.write("</div>");
+		}
+
+		jspWriter.write("</div>");
+
+		if (!ListUtil.isEmpty(_actionDropdownItems)) {
+			jspWriter.write("<div class=\"dropdown\">");
+			jspWriter.write("<div class=\"dropdown-toggle component-action\">");
+			jspWriter.write("<svg class=\"lexicon-icon lexicon-icon-user\" ");
+			jspWriter.write("role=\"presentation\" viewBox=\"0 0 512 512\">");
+			jspWriter.write("<use xlink:href=\"");
+			jspWriter.write(_spritemap);
+			jspWriter.write("#ellipsis-v\" /></svg>");
+			jspWriter.write("</div>");
+			jspWriter.write("</div>");
+		}
+
+		jspWriter.write("</div>");
+		jspWriter.write("</div>");
+		jspWriter.write("</div>");
+
+		return SKIP_BODY;
+	}
+
+	private void _populate() {
+		if (_actionDropdownItems == null) {
+			setActionDropdownItems(_userCard.getActionDropdownItems());
+		}
+
+		if (getCssClass() == null) {
+			setCssClass(_userCard.getCssClass());
+		}
+
+		if (getData() == null) {
+			setData(_userCard.getData());
+		}
+
+		if (getDefaultEventHandler() == null) {
+			setDefaultEventHandler(_userCard.getDefaultEventHandler());
+		}
+
+		if (getCssClass() == null) {
```

Comment:
@markocikos this is how I resolved this (step 3)

Reply:
@carloslancha I think you are still missing getter, setter and handling (overwrite priority behavior) of `elementClasses` in `UserCardTag`.

Reply:
That's done already in `BaseContainerTag`, that's why we don't need to do anything else here, just deprecate `elementClasses` attribute in the tld, which I'm making in this pr.

Reply:
Yep, you are right, there is already a getter and setter :+1: 

But, aren't we still missing overwrite behavior:
```
<clay:user-card-v2
    cssClass="foo1"
    elementClasses="bar1"
    userCard="<%= new UserCard(// cssClass set to foo2, elementClasses set to bar2) %>"
/>
```
What happens? :shrug:  Are `elementClasses` ignored, merged or overwritten?

Maybe this is such an edge case that is not worth dealing with it.